clang/include/clang/Basic/DiagnosticASTKinds.td

```diff
@@ -478,16 +478,24 @@ def warn_odr_function_type_inconsistent : Warning<
   "external function %0 declared with incompatible types in different "
   "translation units (%1 vs. %2)">,
   InGroup<ODR>;
+def err_odr_attr_inconsistent : Error<
+  "attribute %0 is incompatible%select{| in different translation units}1">;
```

Comment:
I can't find `is incompatible` in any of the tests.

Reply:
Good catch, this is dead code and can be removed